.vscode-test.mjs

```diff
@@ -8,27 +8,34 @@ import fs from 'fs';
 import path from 'path';
 
 // Discover test suite folders in src/test/integration
-const BASE_SRC_PATH = './src/test/integration';
+const BASE_SRC_PATH = './src/test/fixtures';
 const BASE_OUT_PATH = './out/test/integration';
 
-const testSuiteFolderNames = fs.readdirSync(BASE_SRC_PATH, { withFileTypes: true })
-  .filter(entry => entry.isDirectory()) // only directories ...
-  .filter(entry => fs.existsSync(path.join(BASE_SRC_PATH, entry.name, "workspace"))) // ... that contain a workspace folder are valid
-  .map(entry => entry.name);
```

Comment:
With this removed, we should update `src/test/integration/README.md` as it is describing the now removed behaviour 🤓

Reply:
I'm splitting up the PR and dealing with the integration tests separately. Moving the folders is introducing test case differences that are exceeding the remit of this PR.